addons/html_builder/static/tests/images.test.js

```diff
@@ -0,0 +1,24 @@
+import { expect, test } from "@odoo/hoot";
+import { animationFrame, dblclick, queryOne } from "@odoo/hoot-dom";
+import { defineWebsiteModels, openSnippetsMenu, setupWebsiteBuilder } from "./helpers";
+
+defineWebsiteModels();
+
+const base64Img =
+    "data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUA\n        AAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO\n            9TXL0Y4OHwAAAABJRU5ErkJggg==";
+
+test("double click on Image", async () => {
+    await setupWebsiteBuilder(`<div><img class=a_nice_img src='${base64Img}'></div>`);
+    await openSnippetsMenu();
```

Comment:
check if you don t have .modal-content before. You don t have a specific class on your dialog ?

You have an issue in your test, if dblclick open another dialog for strange reason you don t know